courses/machine_learning/deepdive2/structured/solutions/4_bqml_linear_benchmark_babyweight.ipynb

```diff
@@ -0,0 +1,629 @@
+{
```

Comment:
<p>This seems like a lot of computational overhead for just finding the average weight. Why not calculate it out like in the first benchmark lab?</p><p><a href="https://github.com/GoogleCloudPlatform/training-data-analyst/blob/master/courses/machine_learning/deepdive/01_bigquery/c_extract_and_benchmark.ipynb" target="_blank">https://github.com/GoogleCloudPlatform/training-data-analyst/blob/master/courses/machine_learning/deepdive/01_bigquery/c_extract_and_benchmark.ipynb</a></p><p>(If you want to stay in sql land, AVG would take on hundredth the time)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/668/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>This is using linear regression with the base features as a benchmark, not the average weight_pounds.</p>

Reply:
Ah whoops, sorry, good push back. I blame my Friday afternoon brain :)